tap_quickbooks/client.py

```diff
@@ -71,12 +158,11 @@ def _write_config(self, token):
         with open(self.config_path, 'w') as file:
             json.dump(config, file, indent=2)
 
-    # Backoff th request for 5 times when Timeout error occurs
+    # Backoff the request for 5 times when Timeout error occurs
     @backoff.on_exception(backoff.expo, Timeout, max_tries=5, factor=2)
     @backoff.on_exception(backoff.constant,
                           (Quickbooks5XXException,
                            Quickbooks4XXException,
-                           QuickbooksAuthenticationError,
                            requests.ConnectionError),
                           max_tries=3,
```

Comment:
Comment says 5 times backoffing the request but max_tries=3. These should be matching.